python/ceed_vector.py

```diff
@@ -58,6 +58,17 @@ def __str__(self):
 
         return out_string
 
+    # Copy the array from a vector into self
+    def copy(self, vec_source):
```

Comment:
I'd change the method name to copy_from, but otherwise I think this is ready to merge!